### PR TITLE
Add e2e smoke-test for YAML verification

### PR DESCRIPTION
**Pull request by vvfesik**

### CODE REVIEW

File: client/cypress/e2e/smoke/yamlToDashboard.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+describe('Localhoste upload yaml to dashboard', function() {
+  beforeEach(() => cy.fixCypressSpec(__filename));
+
+  it('Visits dashboard and uploads Localhoste YAML', function() {
+    cy.visit(`https://dashboard.${Cypress.env('DOMAIN')}`);
+    cy.url().should('contain', `auth.${Cypress.env('DOMAIN')}`);
+
+    cy.get('#username').type(Cypress.env('EMAIL'));
+    cy.get('#password').type(Cypress.env('PASS'), { parseSpecialCharSequences: false });
```
Comment (aslepakurov) [inline client/cypress/e2e/smoke/yamlToDashboard.spec.js:9]: Please change envvar name to this format - `VGS_DASHBOARD_<param>` for clarity

File: .circleci/config.yml [hunk 2]
```diff
@@ -15,10 +15,36 @@ jobs:
...
+          path: /root/app/client/cypress/screenshots/
 
 
 workflows:
   version: 2
-  workflow:
+  build_and_test:
     jobs:
       - build
+  periodical_smoke_test:
```
Comment (aslepakurov) [inline .circleci/config.yml:41]: Please add comment with test description

File: client/cypress/e2e/smoke/yamlToDashboard.spec.js (hunk 1 above)
Comment (vvfesik) [reply client/cypress/e2e/smoke/yamlToDashboard.spec.js]: ✔️

File: .circleci/config.yml (hunk 2 above)
Comment (vvfesik) [reply .circleci/config.yml:41]: ✔️

**Pull request merged**